src/libraries/System.Private.CoreLib/src/System/Half.cs

```diff
@@ -605,32 +607,170 @@ public bool TryFormat(Span<char> destination, out int charsWritten, [StringSynta
         /// <summary>Explicitly converts a <see cref="float" /> value to its nearest representable half-precision floating-point value.</summary>
         /// <param name="value">The value to convert.</param>
         /// <returns><paramref name="value" /> converted to its nearest representable half-precision floating-point value.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
This looks like fairly large method (even after optimizations). Is it really worth it to aggressive inline it?

Reply:
I updated the code so that it won't branch by supported instructions anymore.  
However, even after such modification, the operator is not getting inlined without `[MethodImpl(MethodImplOptions.AggressiveInlining)]` even though the benchmark code makes it hot.  
It turns out that the non-inlined operators are significantly slower than the inlined operators in the loop.

Reply:
Right, you can always make microbenchmarks faster by AggressiveInlining. The code bloat introduced by AggressiveInlining is not necessarily good trade-off for real world code performance.